mtags/src/main/scala-3/scala/scala/meta/internal/pc/ScalaPresentationCompiler.scala

```diff
@@ -186,13 +195,17 @@ case class ScalaPresentationCompiler(
     )
   }
 
-  // TODO NOT IMPLEMENTED
   def documentSymbols(
       params: VirtualFileParams
   ): CompletableFuture[ju.List[DocumentSymbol]] = {
-    CompletableFuture.completedFuture(
-      List.empty[DocumentSymbol].asJava
-    )
+    compilerAccess.withInterruptableCompiler(
+      List.empty[DocumentSymbol].asJava,
+      params.token
+    ) { access =>
+      val driver = access.compiler()
+      driver.run(params.uri, params.text)
```

Comment:
here we're running the PC: I've tried parsing the text directly but then I got all `-1` for the positions...